src/Plugin/search_api/backend/SearchApiElasticsearchBackend.php

```diff
@@ -43,6 +43,11 @@ class SearchApiElasticsearchBackend extends BackendPluginBase implements PluginF
   use PluginFormTrait;
 
   /**
+   * Auto fuzziness setting.
```

Comment:
What does "auto" mean? Are there upstream docs we can link to?

Reply:
Why yes, yes there are. Will update.